lib/v2/models/token_model.dart

```diff
@@ -0,0 +1,30 @@
+class TokenModel {
+  final String chainName;
+  final String contract;
+  final String symbol;
+  final String name;
+  const TokenModel({
+    required this.chainName,
+    required this.contract,
+    required this.symbol,
+    required this.name,
+    });
+}
+
+const SeedsToken = TokenModel(
```

Comment:
What do you think about making `TokenModel` and abstract class and have all the other tokens implement this abstract class?

Reply:
I don't see a reason to have it be abstract

However, we will need to have an abstract class representing a balance with a title if we are going to show planted balance on the cards (as per UX spec)

In that case Token would implement that new abstract class, or it would have to be abstracted in the view model

Not sure the planted balance fits well on the cards since it's not a token, and send/receive don't work the way they work for others. I am thinking it's probably better to have a separate UX for planted.